roles/matrix_mult/tasks/main.yml

```diff
@@ -0,0 +1,87 @@
+---
+- name: Ensure that NFD found nodes with GPU labels
+  # label list should be in sync with:
+  # https://github.com/NVIDIA/gpu-operator/blob/master/pkg/controller/clusterpolicy/state_manager.go#L26
+  shell:
+    (   oc get nodes -oname --ignore-not-found=false -l feature.node.kubernetes.io/pci-10de.present
```

Comment:
See [roles/gpu_operator_run_gpu-burn/tasks/main.yml](https://github.com/openshift-psap/ci-artifacts/blob/master/roles/gpu_operator_run_gpu-burn/tasks/main.yml) for hints on how to solve these ansible-lint errors